crates/bevy_pbr/src/lib.rs

```diff
@@ -121,7 +121,7 @@ impl Plugin for PbrPlugin {
                     .after(SimulationLightSystems::UpdateDirectionalLightFrusta)
                     .after(SimulationLightSystems::UpdatePointLightFrusta)
                     // NOTE: This MUST be scheduled AFTER the core renderer visibility check
-                    // because that resets entity ComputedVisibility for the first view
+                    // because that resets entity Visib for the first view
```

Comment:
typo

Reply:
Yup, but I'm not sure what to replace this text with

Reply:
Ah, these changes remove the dependency I think. The dependency was that `check_visibility()` did `ComputedVisibility.is_visible = false` for every mesh, and `check_light_mesh_visibility()` updated `ComputedVisibility.is_visible` for those meshes that needed to be marked as visible due to being visible by a light. But now that the per-view VisibleEntities define which entities are visible and need extracting, and `check_visibility()` and `check_light_mesh_visibility()` operate on different views, and you take the set union of those for extraction, there is no global computed visibility state to create the dependency. So this can be removed.